src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -11,12 +11,10 @@
     <ProjectGuid>{8772BC91-7B55-49B9-94FA-4B1BE5BEAB55}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableWinRT Condition="'$(TargetGroup)' == 'netcore50'">true</EnableWinRT>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
```

Comment:
I'm fine with removing it. I guess it makes sense if we aren't actually shipping the facade that we don't need to build it. But if we do remove it, we should also delete the "TargetingPackReference" ItemGroup at the bottom of the file (that is only used for desktop builds)